firmware/integration/rusefi_config.txt

```diff
@@ -1052,7 +1057,42 @@ custom maf_sensor_type_e 4 bits, S32, @OFFSET@, [0:1], @@maf_sensor_type_e_enum@
 
 	output_pin_e[LUA_PWM_COUNT iterate] luaOutputPins
 
-	int[57] unusedAtOldBoardConfigurationEnd;;"units", 1, 0, -20, 100, 0
+	#define tcu_role_e_enum "Park", "Neutral" , "Park/Neutral", "Reverse", "Drive", "Manual", "Low", "Upshift", "Downshift", "3", "2", "1"
+	custom tcu_role_e 1 bits, U08, @OFFSET@, [0:3], @@tcu_role_e_enum@@
+	tcu_role_e	tcuState0Role;
```

Comment:
should this be an array?

Reply:
Is this not an array?

Reply:
an array would look like `tcu_role_e [8 iterate] tcuStateRole;`

Reply:
oh right. No, it's not supposed to be an array.

Reply:
Why not? BTW adding the word "iterate" makes it spit out non-array for TS, but an array for the c++ part.

Reply:
oh I am an idiot